chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1267,6 +1267,184 @@
 						"description": "Expect Listing Basic response to have svapResultsUrl"
 					},
 					"response": []
+				},
+				{
+					"name": "PUT /{certifiedProductId}- 400 status and error on adding removed/invalid Test Tools to 2015 listings",
```

Comment:
This should be PUT: /certified_products/{certifiedProductId} - ......
Same for other request

Reply:
Done